src/PockyBot.NET/Persistence/Repositories/PockyUserRepository.cs

```diff
@@ -18,24 +18,23 @@ public PockyUserRepository(DatabaseContext context)
         public PockyUser GetUser(string userId)
         {
             return _context.PockyUsers
-                .Where(x => x.UserId == userId)
                 .Include(x => x.PegsGiven)
                     .ThenInclude(x => x.Receiver)
                 .Include(x => x.PegsReceived)
                 .Include(x => x.Location)
                 .Include(x => x.Roles)
-                .SingleOrDefault();
+                .SingleOrDefault(x => x.UserId == userId);
         }
 
         public List<PockyUser> GetUsersByUsername(string username)
         {
             return _context.PockyUsers
-                .Where(x => x.Username == username)
                 .Include(x => x.PegsGiven)
-                .ThenInclude(x => x.Receiver)
+                    .ThenInclude(x => x.Receiver)
                 .Include(x => x.PegsReceived)
                 .Include(x => x.Location)
                 .Include(x => x.Roles)
+                .Where(x => x.Username == username)
```

Comment:
and this one?

Reply:
Agreed on this one too, we should modify the PR to just include the databasecontext change